ReactAndroid/src/androidTest/js/ScrollViewTestModule.js

```diff
@@ -98,24 +95,45 @@ var ScrollViewTestApp = createReactClass({
         onScroll={this.onScroll}
         onScrollBeginDrag={this.onScrollBeginDrag}
         onScrollEndDrag={this.onScrollEndDrag}
```

Comment:
I don't think there's a reason for these functions to be attached to the component instance. So, after we move the functions out of the `ScrollViewTestApp` class, lets change this to:

```javascript
  <Item
    // ...
    onScroll={onScroll}
    onScrollBeginDrag={onScrollBeginDrag}
    onScrollEndDrag={onScrollEndDrag}
   // ...
```